sql/core/src/test/scala/org/apache/spark/sql/CollationStringExpressionsSuite.scala

```diff
@@ -17,13 +17,11 @@
 
 package org.apache.spark.sql
 
-import scala.collection.immutable.Seq
-
```

Comment:
let's avoid unnecessary changes

Reply:
It will not pass the tests because of unused import

Reply:
are you sure that's the reason? If so, go ahead and remove it
but I do wonder why this only comes up now, it doesn't have to do with your changes

Reply:
Reverted the change